### PR TITLE
Many bug fixes, resizing, video rendering and cleaned up animation code

### PR DESCRIPTION
**Pull request by mfuechec**

### CODE REVIEW

File: client/App.jsx [hunk 1]
```diff
@@ -172,6 +233,9 @@ React.Component {
                         <MainImage videoEmbed={this.state.videoEmbed} onClose={this.onClose.bind(this)} previousImage={this.state.previouslySelectedImageNumber} numOfImgs={this.state.numOfImgs} onScroll={this.onClickArrow.bind(this)} onClick={this.onClickMain.bind(this)} selectedItemId={this.state.selectedItem} imgNum={this.state.imageNumber}/>
                     </div>
                 </div>
+                <div id='m_buyItemModal'>
+                    <BuyItemModal itemName={this.state.itemName} selectedItemId={this.state.selectedItem}></BuyItemModal>
+                </div>
```
Comment (tmchandler) [inline client/App.jsx]: Closing tag here looks to have slid to the left

File: client/components/BuyItemModal.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```
Comment (tmchandler) [inline client/components/BuyItemModal.jsx]: You'll want to clean up the general formatting of this file.  The spans on line 15 and 16 are a bit tangled with 3 different elements all on one line.  The `d=path` values on them will just end up on their own line, but you need to expand out the element tags on the rest of this for readability.

File: client/components/SideImages.jsx [hunk 3]
```diff
@@ -4,16 +4,18 @@ function SideImages(props) {
     const items = [];
     for (var i = 1; i < props.numOfImgs + 1; i++) {
         if (i === 2 && props.videoThumb !== null) {
-            var videoThumbnail = props.videoThumb;
             items.push(
-                <img key='video' className='side video' src={videoThumbnail} onClick={props.onVideoClick} />
-            );
+                <span className='videoThumbContainer'>
+                    <img key='video' className='side video' src={props.videoThumb} onClick={props.onVideoClick} />
```
Comment (tmchandler) [inline client/components/SideImages.jsx]: keys will generally want to be different for each specific item.  I'd change these and just add a number on to them (the current i or something) to make them unique.

File: config.js [hunk 4]
```diff
@@ -1,3 +1,3 @@
 module.exports = {
```
Comment (tmchandler) [inline config.js]: O NOOOOOO!! Remove remove!

File: client/App.jsx (hunk 1 above)
Comment (mfuechec) [reply client/App.jsx]: Nah, BuyItemModal is in a separate div from the other two.

File: client/components/BuyItemModal.jsx (hunk 2 above)
Comment (mfuechec) [reply client/components/BuyItemModal.jsx]: Done

File: client/components/SideImages.jsx (hunk 3 above)
Comment (mfuechec) [reply client/components/SideImages.jsx]: Done

File: config.js (hunk 4 above)
Comment (mfuechec) [reply config.js]: Good point

**Pull request merged**